stable_baselines3/common/utils.py

```diff
@@ -535,3 +545,66 @@ def compat_gym_seed(env: GymEnv, seed: int) -> None:
     else:
         # VecEnv and backward compatibility
         env.seed(seed)
+
+
+def compat_gym_26_step(
+    step_returns: Union[GymStepReturn, Gym26StepReturn],
+    warn=True,
+) -> Gym26StepReturn:
+    """
+    Transform step returns to old step API irrespective of input API.
+    This makes env written with both gym < 0.26 and gym > 0.26
+    compatible with SB3.
+    This is a simplified version of the helper found in Gym.
+
+    :param step_returns: Items returned by `env.step()`.
+        Can be (obs, reward, done, info) or (obs, reward, terminated, truncated, info)
+    :param warn: Whether to warn or not the user
+    :return: (obs, reward, done, info) with info["TimeLimit.truncated"] = truncated
+    """
+    if len(step_returns) == 4:
+        if warn:
+            warnings.warn(
+                "You are using gym API < 0.26, please upgrade to gym > 0.26 "
+                "step API with 5 values returned (obs, reward, terminated, truncated, info) "
+                "instead of the current 4 values (obs, reward, done, info). "
+                "Please read Gym documentation for more information.",
+                DeprecationWarning,
+            )
+        observations, rewards, done, infos = step_returns
+        truncated = infos.get("TimeLimit.truncated", False) # pytype: disable=attribute-error]
+        return observations, rewards, done, truncated, infos # pytype: disable=bad-return-type
+    else:
+        assert len(step_returns) == 5, f"The step function returned {len(step_returns)} values instead of 5"
+        return step_returns # pytype: disable=bad-return-type
+
+
+def compat_gym_26_reset(
+    reset_return: Union[GymObs, Tuple[GymObs, Dict]],
+    warn: bool = True,
+) -> Tuple[GymObs, Dict]:
+    """
+    Transform gym reset return to new API (gym > 0.26)
+
+    :param reset_return: (obs) or (obs, info)
+    :param warn: Whether to warn or not the user
+    :return: (obs, info)
+    """
+    if isinstance(reset_return, tuple):
+        assert len(reset_return), (
```

Comment:
should be `assert len(reset_return) == 2`?

Reply:
it should be yes, but the code is not longer used.